<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatRooms</class>
 <widget class="QWidget" name="ChatRooms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:color #2c698d;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Chat Room Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtChatRoomname">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:color #fff;
border-radius: 15px;
  border: 2px solid #fff;</string>
      </property>
      <property name="placeholderText">
       <string>Chat Room</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/resources/images/log-out.svg</normaloff>:/resources/images/log-out.svg</iconset>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ChatterBox Users</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listUsers">
        <property name="styleSheet">
         <string notr="true">background:color #fff;
border-radius: 15px;
border: 2px solid #fff;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listAllRoomUsers">
        <property name="styleSheet">
         <string notr="true">background:color #fff;
border-radius: 15px;
border: 2px solid #fff;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnSaveChatRoom">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:color #1AC0C6;
border-radius: 15px;
  border: 2px solid;
  padding: 10px;
  width: 200px;
  height: 150px;</string>
      </property>
      <property name="text">
       <string>Save Chat Room</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
